core/new-gui/src/app/workspace/component/result-panel/console-frame/console-frame.component.ts

```diff
@@ -106,5 +122,13 @@ export class ConsoleFrameComponent implements OnInit, OnChanges {
 
   displayConsoleMessages(operatorId: string) {
     this.consoleMessages = operatorId ? this.workflowConsoleService.getConsoleMessages(operatorId) || [] : [];
+    console.log("got messages", this.consoleMessages);
+  }
+
+  workerIdToAbbr(workerId: string) {
```

Comment:
please annotated explicit return type to all functions, including some existing functions above that are not anootated

Reply:
done.